### ISSUE TITLE
There's a blank window bug on Windows with headless Chromium 129

### ISSUE THREAD

**Issue by mdmintz** [seleniumbase/SeleniumBase#3156]
## There's a blank window bug on Windows with headless Chromium 129

There's already a ticket here: https://issues.chromium.org/issues/367755364.
(It doesn't sound like they're going to fix it, so I will.)

The simple fix is to move the window off-screen when using headless Chromium on Windows:

`--window-position=-2400,-2400`

```python
chrome_options.add_argument("--window-position=-2400,-2400")
```

That's what I've been telling people to do on Stack Overflow:
[Stack Overflow: Chrome 129 Headless shows blank window](https://stackoverflow.com/a/78999088/7058266)
[Stack Overflow: Selenium headless broke after Chrome update](https://stackoverflow.com/a/79004612/7058266)

I'll apply my fix directly into SeleniumBase so that it's automatic when using headless Chromium on Windows.

**Comment by CatalinaCharlotte** [comment]
I have a question: If I am using 'Driver()', how should I add 'option'?

**Comment by mdmintz** [comment]
This was resolved in `4.31.0` - https://github.com/seleniumbase/SeleniumBase/releases/tag/v4.31.0
